routes/feedback-thanks/feedback-thanks.controller.js

```diff
@@ -6,7 +6,8 @@ module.exports = (app, route) => {
   route.draw(app)
     .get((req, res) => {
       res.render(name, routeUtils.getViewData(req, {
-        title: res.__('feedback-thanks.text'),
+        siteUrl: req.protocol + '://' + req.get('host'),
```

Comment:
Maybe it would be better to set this globally instead of in each controller?

Ie, you could load a middleware in app.js that attaches it to app.locals, then it'll be available in all views. Something like this should work:

```
app.use((req, res, next) => {
  app.locals.siteUrl = req.protocol + '://' + req.get('host')
  next()
})
```

Then in views it would be available as simply `siteUrl`, or in controllers it would be available at `app.locals.siteUrl`

Reply:
ah yes! I was fiddling with that, but my brain forgot the syntax. Will do momentarily!